src/main/java/com/spaceclub/global/mail/MailInfo.java

```diff
@@ -0,0 +1,11 @@
+package com.spaceclub.global.mail;
+
+public record MailInfo(
+        String[] address,
+        String[] ccAddress,
```

Comment:
List의 부가적인 기능이 유용할거라 String[]대신 List<String>으로 받는 것을 추천드려요
```suggestion
        List<String> address,
        List<String> ccAddress,
```

Reply:
static factory method로 해결 완료